docs/src/app/components/pages/components/SVGIcon/README.md

```diff
@@ -0,0 +1,15 @@
+## SVG Icons
+Using mui.SvgIcon, we can create a custom [svg component](https://www.google.com/design/spec/style/icons.html#). Here we are creating the ActionHome SvgIcon for this docs site, and using it in some separate component.
```

Comment:
https://www.google.com/design/spec/style/icons.html#icons-system-icons is probably better.